CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.9)
```

Comment:
The common runtime still supports older CMAKE versions via cmake_minimum_required(VERSION 3.9...3.31).  If we merge this we break customers on older versions of CMAKE no?

Reply:
We're using the same minimum version: `3.9`. `cmake_minimum_required(VERSION 3.9...3.31)` is setting the minimum to `3.9` and the maximum to `3.31`. I thought about adding that but figured that might cause issues if we forget to bump it in the future.

Reply:
It does not set a maximum version. What it does is that it sets policies defined between 3.9 and 3.31.

Please, please use the `3.9...3.31` or at the very least bump it to any version pinned to an lts distro that you support.